torch/_inductor/codegen/triton.py

```diff
@@ -2279,7 +2285,8 @@ def end_current_reduction_loop():
                         node not in done
                         and fits_in_main_body(other_node)
                         and not (
-                            current_loop_writes & other_node.recursive_predecessors
```

Comment:
Soooo, `current_write_loops` doesn't exist and this code path must already be broken. I glanced at the history and made a guess that the name should have changed to `current_loop_reduced_writes`, but I don't know what the code is doing exactly so I may be wrong. I see that @Chillee made some changes to this file recently. @Chillee, maybe you can vet this?

Reply:
Or maybe @ezyang? I see your name around these parts.

Reply:
Sorry, scratch that. I guess I see @jansel's name most around this section.

Reply:
I think it's defined above, in `generate_node_schedule(self, nodes, numel, rnumel`  Maybe you need `nonlocal current_write_loops`

Reply:
@eellison, sorry my comment has the wrong variable name. My _change_ actually replaces `current_loop_writes` with `current_loop_reduced_writes`. I don't find `current_loop_writes` anywhere in this file.

Reply:
I think this one is on @peterbell10?

Reply:
The PR is reverted  already anyway, so a rebase should fix it,